great-idea/index.html

```diff
@@ -17,7 +17,69 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
-      
+    <header>
```

Comment:
i'd like to see a full-content container here to deal with keeping everything where it should be on desktop views regardless of window width.